src/commands/deployWorkspaceProject/getDefaultValues/getDefaultContextValues.ts

```diff
@@ -58,21 +58,19 @@ export function triggerSettingsOverride(settings: DeployWorkspaceProjectSettings
 }
 
 async function displaySettingsOverrideWarning(context: ISubscriptionActionContext, item: ContainerAppItem | ManagedEnvironmentItem): Promise<void> {
-    let treeItemType: string;
+    let treeItemType: string | undefined;
     if (ContainerAppItem.isContainerAppItem(item)) {
-        treeItemType = 'container app item ';
+        treeItemType = 'container app item';
     } else if (ManagedEnvironmentItem.isManagedEnvironmentItem(item)) {
-        treeItemType = 'container environment item ';
-    } else {
-        treeItemType = '';
+        treeItemType = 'container environment item';
     }
 
     const resourceName: string = parseAzureResourceId(item.id).resourceName;
     await context.ui.showWarningMessage(
-        localize('overrideConfirmation', `Deployment will target {0}"{1}".\n\nAny workspace deployment settings will be skipped.  Would you like to proceed?`, treeItemType, resourceName),
+        localize('overrideConfirmation', `Deployment will target {0} "{1}".\n\nAny workspace deployment settings will be skipped.  Would you like to proceed?`, nonNullValue(treeItemType), resourceName),
         { modal: true },
         { title: localize('continue', 'Continue') }
     );
 
-    ext.outputChannel.appendLog(localize('confirmedOverride', 'User confirmed deployment will target {0}"{1}" instead of existing workspace settings.', treeItemType, resourceName));
+    ext.outputChannel.appendLog(localize('confirmedOverride', 'User confirmed deployment will target {0}"{1}" instead of existing workspace settings.', nonNullValue(treeItemType), resourceName));
```

Comment:
Need to change the spacing here too
```suggestion
    ext.outputChannel.appendLog(localize('confirmedOverride', 'User confirmed deployment will target {0} "{1}" instead of existing workspace settings.', nonNullValue(treeItemType), resourceName));
```